### ISSUE TITLE
Setter for X,Y

### ISSUE THREAD

**Issue by aldari** [xieziyu/angular2-draggable#38]
Hello, 
This is a nice directive.
I would like to have an opportunity to set initial value (or may be change) of a oldPosition property. I am not sure, but this property has values that I want to change.

I want to create a board with text cards, where each card has properties text: string, x,y:number.
I have created a card component and use this directive to change and track its position. I have find how to create dynamically multiple cards. What i miss is to position cards at their initial position on page load after a page reload. I have create save procedure, it saves cards text and coordinates.

**Comment by CLEMARCx** [comment]
Just set the transform(X,Y) css attribute.

**Comment by ctrl-brk** [comment]
css is no good here since it will always be the same. Would be nice to be able to get and set position from the code.

**Comment by aldari** [comment]
I have tried to use transform inside style. This changes the card position, but once you click or drag the card it returns to coordinates according internal oldPosition property.

You can use events to get the position.

**Comment by aldari** [comment]
Hi,

I have downloaded sources of this repository to get a test platform. 
This task can be split into two:
- transferring inside X,Y values.
- changing the position of html block(in my case it is a card)

Fortunately, i found directive source and it is in typescript.
\lib\src\directive\angular-draggable.directive.ts

The first task solved by adding input for x and y
`  @Input() X: number = 0;
  @Input() Y: number = 0;`

The second was a little bit complex to find a proper values of all properties. So i just call methods from mouse events. 
I have replaced ngInit:
`    if (this.allowDrag) {
      let element = this.handle ? this.handle : this.el.nativeElement;
      this.renderer.setElementClass(element, 'ng-draggable', true);
    }`

`    if (this.allowDrag) {
      let element = this.handle ? this.handle : this.el.nativeElement;
      this.renderer.setElementClass(element, 'ng-draggable', true);
    }
    
    this.orignal = this.getPosition(0, 0);
    this.pickUp();
    this.moveTo(this.X, this.Y);
    this.putBack();`
